src/UnitySampleProject/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -15,6 +24,6 @@ EditorBuildSettings:
     path: Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scenes/SampleImportNftTexture.unity
     guid: d31091c4b9c13af48af91512a5127882
   - enabled: 1
-    path: Assets/Samples/web3.unity SDK/2.5.0/LootBoxes/LootBoxes.unity
+    path: Packages/io.chainsafe.web3-unity.lootboxes/Samples/SampleLootboxes.unity
```

Comment:
if this scene is required to be in Editor Build Settings (build index is referenced in script), we should add it during sample import, take a look at `SampleInitializer.cs`